js/main.js

```diff
@@ -35,6 +42,18 @@ Site = {
   onResize: function() {
     var _this = this;
 
+    _this.resizeDelay();
+  },
+
+  resizeDelay: function() { 
```

Comment:
Im not 100% sure that is the correct reference to the timeout as the scope of the var looks weird. It could well be fine but if Paul Irish [aka god] wrote something I'd generally prefer that over my own code